src/coreclr/debug/createdump/createdump.h

```diff
@@ -91,21 +90,34 @@ typedef int T_CONTEXT;
 #include <array>
 #include <string>
 
+enum DumpType
```

Comment:
`enum class`

Reply:
What does an enum class give us?

Reply:
Not contaminating the namespace, and some type safety